tests/Functional/PlanCommandsTest.php

```diff
@@ -67,29 +67,18 @@ public function testPlanInfoCommand()
      * @covers \Pantheon\Terminus\Commands\Plan\SetCommand
      *
      * @group plan
-     * @group long_fixme
+     * @group long
      */
     public function testSetPanCommand()
     {
-        // Get the current site plan.
-        $plan = $this->terminusJsonResponse(sprintf('plan:info %s', $this->getSiteName()));
-        $this->assertArrayHasKey('sku', $plan);
-        $this->assertNotEmpty($plan['sku']);
-        $currentPlanSku = $plan['sku'];
+        $plans = $this->terminusJsonResponse(sprintf('plan:list %s', $this->getSiteName()));
+        $smallPlans = array_filter($plans, fn ($plan) => false !== strpos($plan['sku'], 'small'));
```

Comment:
My editor is flagging this line as a syntax error, although `php -l` says there are no errors.  I was also unfamiliar with `fn()` and this expression was confusing to me: `fn ($plan) => false !== strpos($plan['sku'], 'small')`

Searching the PHP documentation for `fn` doesn't find useful results: https://www.php.net/manual-lookup.php?pattern=fn&scope=quickref

Could we add a comment with a link to the relevant documentation?
```
// fn() is an arrow function: https://www.php.net/manual/en/functions.arrow.php
```

Reply:
@namespacebrian Arrow functions support started at 7.4 https://www.php.net/manual/en/functions.arrow.php 
It seems your IDE is setup for earlier php version by default

Reply:
Yes, I linked that same page in my comment above.

Reply:
I'm requesting that a link to that page be included in a comment above the line.

Reply:
why?

Reply:
To clarify for anyone else reading the code who had the same confusion.  It's often helpful to include a comment above code that may be confusing to someone else reading the code.

Reply:
I don't think we need to add this. This is a basic PHP feature.

Reply:
What's the harm in adding it?

Reply:
Unnecessary comments

Reply:
I mean, I think that the code does not require such obvious commenting, otherwise we should accompany with comments every  single `??` (Null Coalescing Operator).

Reply:
I agree that comments should explain program behavior, not language features.

Reply:
Okay, thanks. I was going to ask for input about it before carrying the discussion further.